network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -368,6 +368,11 @@ class ProductPage(AirtableMixin, FoundationMetadataPageMixin, Page):
         related_name='votes',
     )
 
+    override_translatable_fields = [
+        SynchronizedField('votes'),
+        SynchronizedField('creepiness_value'),
+    ]
+
```

Comment:
I remember you saying we no longer need this, @TheoChevalier?

Reply:
correct

Reply:
```suggestion
```